recipes/htseq/0.6.1/meta.yaml

```diff
@@ -12,6 +12,8 @@ package:
   version: 0.6.1
 
 requirements:
+  build:
```

Comment:
Build number increase in this recipe please.

Reply:
No changes to that recipe.

Reply:
Oh, wait, I see there is one... Sorry. I'm actually not clear if that's needed. I put that in attempting to fix `circleci build` errors, but I never got that working.